ModuleConfig.cfc

```diff
@@ -12,17 +12,16 @@ component {
 	this.webURL                     = "http://www.ortussolutions.com";
```

Comment:
Make sure you add our `cfformat` rules and run them.

Reply:
done